### ISSUE TITLE
Should PolylineVertex have a Bulge property ?

### ISSUE THREAD

**Issue by GogoVux** [haplokuon/netDxf#279]
Hello,
i have got some dxf files containing POLYLINE entities with VERTEX entities having nonzero bulge value (code 42).
The PolylineVertex class does not have a Bulge property, but the Vertex and LwPolylineVertex classes have.
Are the bulge values lost at read ?
How can i read, or create a POLYLINE entity with arc segments in netDxf  ?
For some reason ist must be a POLYLINE, not a LWPOLYLINE.

Thanks, also for that great library.

**Comment by haplokuon** [comment]
No, only the vertices of 2d polylines can have a bulge value. The POLYLINE structure of the DXF not only holds information about 3d polylines and polyface meshes, but long time ago, previous to 2000, they use to hold also the data for 2d polylines. The old 2d polyline was superseded by the more memory efficient LWPOLYLINE.
